src/careers/_template-careers-related-links.html

```diff
@@ -1,11 +1,10 @@
-<div class="block
-            block__bg
-            block__border-top
-            block__border-right
-            block__flush-top
-            block__flush-bottom
-            block__flush-sides">
-    <div class="">
+<section class="block
+                block__bg
+                block__border-top
+                block__border-right
+                block__flush-top
+                block__flush-bottom
+                block__flush-sides">
             {# TODO: Add Office of Civil Rights URL. #}
```

Comment:
Looks like TODO can be removed with the addition of https://github.com/cfpb/cfgov-refresh/pull/972/files#diff-17ad66c347931e12a8c76283b40cbb90R15